vmdb/app/models/miq_request_workflow.rb

```diff
@@ -949,7 +949,7 @@ def each_ems_metadata(ems_ci = nil, klass = nil, &_blk)
 
   def get_ems_metadata_tree(src)
     @ems_metadata_tree ||= begin
-      st = Time.now
+      st = Time.zone.now
```

Comment:
You're right, it's not, but rubocop complained about the Time.now in the change further down in the file, so I figured I'd change it here too to be consistent.